src/couldBeSharedComponents/LayoutPanels/LayoutPanels.tsx

```diff
@@ -143,70 +170,81 @@ export const LayoutPanels = ({
     <button
       className={layoutPanelStyles.panelCloseButton}
       onClick={handleLeftToggleButtonClick}
+      aria-label="Toggle left panel"
     >
       {isLeftPanelOpenToUse ? "<" : ">"}
     </button>
   );
   const rightPanelButtonToUse = rightPanelToggleButton
     ? cloneElement(rightPanelToggleButton, {
         onClick: handleRightToggleButtonClick,
+        "aria-label": "Toggle right panel",
       })
     : internalRightPanelCloseButton;
 
   const leftPanelButtonToUse = leftPanelToggleButton
     ? cloneElement(leftPanelToggleButton, {
         onClick: handleLeftToggleButtonClick,
+        "aria-label": "Toggle left panel",
       })
     : internalLeftPanelCloseButton;
 
-  return (
```

Comment:
I realized the existing markup had some nested ternaries which needed to be removed. The diff is weird here.